### DISCUSSION TITLE
Locale description

### DISCUSSION THREAD

**Discussion by 00thirdeye00** [cieslarmichal/faker-cxx#818] [General]
Hej @cieslarmichal, can you describe more about the issue #752.

**Comment by cieslarmichal** [comment]
I wrote a description there in comments :)

**Reply by 00thirdeye00** [reply]
should we have locale in all the modules with languages/countries?

**Reply by cieslarmichal** [reply]
Yes, every time we use some country or language we should migrate to locales
